driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/nanosleep.bpf.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2023 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(nanosleep_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct ringbuf_struct ringbuf;
+        if(!ringbuf__reserve_space(&ringbuf, NANOSLEEP_E_SIZE))
+        {
+                return 0;
+        }
+
+        ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_NANOSLEEP_E);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: interval (type: PT_RELTIME) */
+        struct __kernel_timespec ts = {0};
```

Comment:
minor: Could you rename the variable to `req` to match the parameter as in the syscall docu: `int nanosleep(const struct timespec *req, struct timespec *rem);`